sieve.py

```diff
@@ -752,6 +765,21 @@ def run_batch(project, test, dir, mode, stage, docker):
         log_dir = os.path.join(
             options.log, options.project, options.test, options.stage, options.mode
         )
+        if options.mode == "learn-twice":
```

Comment:
You can define the mode name `learn-twice` and `learn-once` in the common.py in `class sieve_modes:` and avoid using string text here.